src/dashboard/webserver/server.py

```diff
@@ -0,0 +1,92 @@
+from flask import Flask, render_template, g, request
+from src import run_config
+from src.dashboard.webserver.server_utils import get_db, get_neighboring_devices, get_alerts
+from src.database.models import DeviceInformation, EmailInformation, AnomalyEquations
+
+app = Flask(__name__)
+
+
+@app.before_request
+def init_db():
+    g.db = get_db(run_config.db_file)
+
+
+@app.route('/ids-priv/settings/', methods=['GET', 'POST'])
```

Comment:
Is there any way to verify that the request comes from the dashboard? Not functionally a concern but security wise, an IoT device might be able to take itself off of the list of devices being tracked.